docs/contributing/contributing-to-decred.md

```diff
@@ -0,0 +1,43 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Code.svg" /> Contributing to Decred
+
+---
+
+Decred is open to contributions across all domains (including software development, design, communications, community support, research and documentation). At its core, Decred is an open source project with a mission to build free software which provides universal access to a method of securely storing and communicating value.
+
+As with many other open source software development projects, people working on Decred do so autonomously and voluntarily. People contribute to Decred because they want the project to succeed and there is some aspect they are personally attached to, enough to want to spend some of their time working on it. The Treasury has funds to pay people for this work, once they demonstrate that they are capable of adding significant value in a particular domain.
+
+The strength of the open source approach to production is that individuals efficiently allocate themselves to the tasks they are most interested in and apt for. Contributors are typically not keen to spend time managing or directing others, so this model tends to only suit people who enjoy working independently in a self-directed manner.
+
+
+
+---
+
+## <img class="dcr-icon" src="/img/dcr-icons/Introductions.svg" /> Becoming a paid Contractor
+
+Decred has a Treasury fund to pay contributors who work effectively to advance the project. To become a paid contractor one must demonstrate that one's work is of value to the project. This [blog post](https://blog.decred.org/2017/07/25/Decred-Recruiting/) by Project Organizer Jake Yocom-Piatt is still the definitive guide to becoming a contractor.
+
+The people who are best placed to judge whether a new contributor would make a good addition to a team are the people who are already working in that team. New contributors who produce good work will be invited to become a paid contractor.
+
+At present, this process is informal, but a formal process for granting a Decred Contractor Clearance (DCC) that allows an individual to bill for their work has been [proposed](https://proposals.decred.org/proposals/fa38a3593d9a3f6cb2478a24c25114f5097c572f6dadf24c78bb521ed10992a4) and approved by stakeholders. This process will involve three established contractors in the relevant domain agreeing that a new contributor should be invited to become a contractor. This DCC model will go live once the [contractor management system](https://github.com/decred/contractor-mgmt) is operational.
+
+As part of becoming a paid contractor, you will agree an hourly rate at which you can bill for your work on the project. Contractors are also eligible for a portrait to be shown on the [contributors page](https://decred.org/contributors/) of decred.org.
```

Comment:
`agree on hourly rate` or `agree on a hourly rate` I guess?

Reply:
I think `an hourly` is more correct than `a hourly`

Reply:
I think this is fine, could be "agree on an hourly rate" though if that reads better for non native English speakers.